packages/world/mud.config.ts

```diff
@@ -355,10 +364,10 @@ export default defineWorld({
     },
   },
   modules: [
-    {
-      artifactPath: "@latticexyz/world-modules/out/PuppetModule.sol/PuppetModule.json",
-      root: false,
-      args: [],
-    },
+    // {
+    //   artifactPath: "@latticexyz/world-modules/out/PuppetModule.sol/PuppetModule.json",
+    //   root: false,
+    //   args: [],
+    // },
```

Comment:
this broke the dev process trying to update the world, do we currently need it?

Reply:
no can remove it, if we need it can be added back later via a script vs here